enhancements/optional-manifest.md

```diff
@@ -52,20 +52,20 @@ ServiceMonitor and VerticalPodAutoscaler are among the supported resources. They
 
 A resource can be marked as optional by leveraging [Explicit Bundle Properties](https://github.com/operator-framework/enhancements/blob/master/enhancements/properties.md). Therefore a new file can be added to the bundle metadata directory. The file itself can have any name, but it has to be in the top-level metadata directory of the bundle and to consist of YAML with a top-level `properties` element containing an array of type/value tuples. There should be a single file containing a top-level `properties` element.
 
-The property type for optional manifest is `olm.manifests.optional`. Its value field is a `files` structure, which contains a list of optional files.
+The property type for optional manifest is `olm.manifests.optional`. Its value field is a `manifests` structure, which contains a list of optional manifests. These manifests are identified by group/kind/namespace/name. The namespace value is optional. For cluster scoped resources or resources where no namespace is specified in the manifest file the field has to be empty, which leads to two slashes being next to each other.
 
 Example:
 
 ~~~
 properties:
 - type: 'olm.manifests.optional'
   value:
-    files:
-    - verticalpodautoscaler.yaml
-    - servicemonitor.yaml
+    manifests:
+    - 'monitoring.coreos.com/PrometheusRule/mynamespace/myrule'
+    - 'authorization.openshift.io/ClusterRoleBinding//mycrbinding'
```

Comment:
I'd prefer to see explicit group, version, kind, name, and namespace fields for each list item so that:
- authors don't have to remember the correct order to provide them
- there's no ambiguity when dealing with cluster-scoped APIs

Reply:
I have updated the PR for the enhancement proposal and also the PR for the implementation in this sense